docs/Makefile

```diff
@@ -167,3 +167,11 @@ genkeyimg:
 	sed -i '/^\.\. END_LS_PNG/e find _static/keybindings/* | awk '"'"'{ print length, $$0 }'"'"' | sort -n | cut -d" " -f2- | awk '"'"'{print ".. image:: /" $$1}'"'"'' manual/commands/keybindings.rst
 	@echo
 	@echo "Keybinding images have been generated."
+
+genwidgetscreenshots:
+	rm -rf screenshots/widgets
+	@echo "Generating screenshots for widgets using pytest fixtures."
+#    pytest -o python_files="ss_*.py" -o python_functions="ss_*" -q --no-header --no-summary ../test
+	pytest -o python_files="ss_*.py" -o python_functions="ss_*" --backend x11 ../test
```

Comment:
Screenshots are generated by only running pytest on files with the `ss_` prefix.